### PR TITLE
IEFixes, better responsiveness and resizing, remove branding

### PR DESCRIPTION
**Pull request by fredantell**
This is most easily reviewed commit by commit.

There are a handful of changes that allow this to run properly in IE11.
There is also a decent amount of bug fixing to improve the resizing of the chart. Because this was being set programmatically originally there were some strange bugs here.
Also added a copy code snippet button, although that for some reason fails to run in IE11 (while IE11 supports selection.addRange(range) when I run that code it fails to add the range object properly to the current selection).

### CODE REVIEW

File: assets/mr-plotter/js/control.js [hunk 1]
```diff
@@ -304,8 +304,17 @@ function applySettings() {
...
+    // for its own calculations. As a result, these functions will keep
+    // changing the result of each other's next invocation until they
+    // eventually converge.
+    // While it would be preferable to fix this dependency, at the least
+    // we can re-run until the answer stabilizes
+    for (var i = 0; i < 10; i++) {
+        this.idata.TARGETWIDTH = this.idata.widthFunction();        
+        s3ui.updateSize(this, true);
+        if (this.idata.TARGETWIDTH == this.idata.widthFunction()) break;
+    }
```
Comment (shaunlebron) [inline assets/mr-plotter/js/control.js:317]: nice quickfix to wait for the convergence 👍

File: assets/index.html [hunk 2]
```diff
@@ -581,10 +581,22 @@ <h3 style="margin-top:0px;" data-localize="select_streams">Select Streams
...
-                                      // alert("This is not IE ");
+                                      //Load helper libraries and polyfills to deal with IE
+                                      console.log('This is IE');
+                                      var body = document.querySelector('body');
+                                      var ieScripts = [
+                                        "canvg/rgbcolor.js", // helps save as PNG
+                                        "canvg/StackBlur.js", // helps save as PNG
+                                        "canvg/canvg.js", // helps save as PNG
+                                        "innersvg-polyfill/innersvg.js" // helps save as SVG/PNG
+                                      ];
```
Comment (shaunlebron) [inline assets/index.html:559]: IE 💯

File: assets/index.html [hunk 3]
```diff
@@ -463,7 +463,7 @@ <h3 style="margin-top:0px;" data-localize="select_streams">Select Streams
 
                                 </svg>
                                 <div id="plot-directions">
-                                  <img src="images/qs_start_up.jpg" >
+                                  <img src="images/start_up_instructions.png" >
```
Comment (shaunlebron) [inline assets/index.html:466]: 👍

**Comment by shaunlebron** [comment]
LGTM

**Comment by TechauraIO** [comment]
I am going ahead and merging this pull request now.

**Pull request merged**